### DISCUSSION TITLE
kernel panic and unable to get past internet setup

### DISCUSSION THREAD

**Discussion by McGeoMetro0045** [Maccraft123/Cadmium#118] [General]
so i ran ./install-to-emmc when i loaded into the usb through using ctl-u i got to select the wifi network then it brought me to a page with a longer what looks to be a selection box with no options in it, i ended up rebooting and removing the usb after quitting the installer and now i am getting a kernel panic

**Comment by Edosctilde** [comment]
Same problem here, any solution?
